charts/velero/examples/aws/secret.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: velero-aws-creds
+data:
+  # echo -n "[default]
+  # aws_access_key_id=ACCESSKEY
+  # aws_secret_access_key=SECRETACCESSKEY
+  # " | base64
+  cloud: W2RlZmF1bHRdCmF3c19hY2Nlc3Nfa2V5X2lkPUFDQ0VTU0tFWQphd3Nfc2VjcmV0X2FjY2Vzc19rZXk9U0VDUkVUQUNDRVNTS0VZCg==
```

Comment:
See other comment regarding the usage of `stringData`

Reply:
Thanks, done